src/ucp/api/ucp.h

```diff
@@ -1703,9 +1703,9 @@ void ucp_dt_destroy(ucp_datatype_t datatype);
  * subsequent communication operations to the same @ref ucp_worker_h "worker"
  * which follow the call to @ref ucp_worker_fence "fence".
  *
- * @note The primary diference between @ref ucp_worker_fence "ucp_worker_fence()"
+ * @note The primary difference between @ref ucp_worker_fence "ucp_worker_fence()"
  * and the @ref ucp_worker_flush "ucp_worker_flush()" is the fact the fence
- * routine does not gurantee completion of the operations on the call return but
+ * routine does not guarantee completion of the operations on the call return but
  * only ensures the order between communication operations. The
  * @ref ucp_worker_flush "flush" operation on return guarantees that all
```

Comment:
Thanks for fixing this, but I would prefer these changes to go as a separate PR, since it has nothing to do with RoCE.